web/services/getPersonalSummaryByDateService.php

```diff
@@ -129,7 +129,27 @@
         }
         if ($minutes < 10)
             $minutes = "0" . $minutes;
-        $pendingHolidays = $hours . ":" . $minutes;
+
+        $currentJourney = 8;
+        $journeys = UsersFacade::GetUserJourneyHistoriesByIntervals($initDate, $date, $userVO->getId());
```

Comment:
You don't need to query the facade for this data, it's already been fetched in another part of the code: https://github.com/Igalia/phpreport/blob/master/web/tasks.php#L91

Reply:
Forget this comment, my mind was in the wrong layer ;)